configs/offsets/tracker_input.json

```diff
@@ -1,7 +1,7 @@
 {
   "go": {
-    "versions": ">= 1.17",
-    "inspect": "./configs/offsets/std_inspect.go",
+    "versions": ">= 1.18",
```

Comment:
Reminder: We will need to document this and state it clearly in the release notes.

Reply:
Adding the breaking change label, as I understand that we won't support Go 1.17 anymore, right?

Reply:
I fixed the go-offsets tracker to support custom files for modules, so we can now bring back Go 1.17, no issues.